### DISCUSSION TITLE
Doubt

### DISCUSSION THREAD

**Discussion by Rishabh-Ctrl18** [prbhtkumr/PhoneSploit#34] [General]
What can be done if it shows 'adb' is not recognized as an internal or external command?

**Comment by prbhtkumr** [comment]
you can try extracting the `adb.rar` file.

**Reply by Rishabh-Ctrl18** [reply]
I did it but not working, I'll try again. 
Is there any other particular reason for system not able to recognize it?

**Reply by prbhtkumr** [reply]
windows basically looks for a file to execute in the current directory and in the path. the reason it says not recognized is because it can't find it in either of them.

**Reply by Rishabh-Ctrl18** [reply]
okay I'll see over that
